modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -95,15 +95,17 @@ Runtime Fabric on Self-Managed Kubernetes supports any ingress controller that i
 For GKE customers, the ingress controller included with GKE provisions a separate HTTP load balancer per application by default. Before using the ingress controller provided by GKE, learn more about its behavior, exploring workarounds, or using another ingress controller if this behavior is undesirable. See link:https://help.mulesoft.com/s/article/Default-Ingress-Controller-Behavior-with-Runtime-Fabric-on-GKE[KB article] for more details.
 ====
 
-=== External Log Forwarding
+=== Logs
 
-Runtime Fabric on Self-Managed Kubernetes does not include external log forwarding. However, you can use any external log forwarding agent that is compatible with your Kubernetes environment running on Amazon EKS, AKS, or GKE. Common log forwarding agents include:
+To retrieve logs for a Mule application deployed in a Docker container on Runtime Fabric on Self-Managed Kubernetes, send them to `stdout`. Refer to the Docker logging documentation for details.  
```

Comment:
Applications deployed on Runtime Fabric on Self-Managed Kubernetes direct logs to `stdout`. The container runtime traps these logs and writes them to a file. Where these files are stored depends on the container runtime and configuration. Refer to your Kubernetes provider documentation for details.

Reply:
edited